### PR TITLE
PW-2346: add brand and networkPaymentReference fields to support Netw…

### LINKED ISSUE #338: [PW-2346] missing "brand" field in DefaultPaymentMethodDetails
**Describe the bug**
I'm trying to follow this [doc](https://docs.adyen.com/checkout/network-tokenization#existing-network-tokens), but there is no way to set paymentMethod.brand. 

**How to reproduce**
N/A

**Expected behavior**
should have a field to set paymentMethod.brand

**Desktop (please complete the following information):**
 - Java Version: [11.0.2]
 - Library Version: [5.0.0]

**Additional context**

### PR DESCRIPTION
**Pull request by Aleffio**
…ork Tokenization, fixes #338

**Description**
support network tokenization: https://docs.adyen.com/checkout/network-tokenization#existing-network-tokens

**Tested scenarios**

**Fixed issue**:  #338

### CODE REVIEW

File: src/test/java/com/adyen/CheckoutTest.java [hunk 1]
```diff
@@ -1288,6 +1301,33 @@ protected PaymentsRequest createEncryptedPaymentsCheckoutRequest() {
...
+
+        paymentsRequest.setReference("Your order number");
+        paymentsRequest.setAmount(createAmountObject("USD", 1000L));
+        DefaultPaymentMethodDetails defaultPaymentMethodDetails = new DefaultPaymentMethodDetails();
+        defaultPaymentMethodDetails.setType("networkToken");
+        defaultPaymentMethodDetails.setBrand("visa");
+        defaultPaymentMethodDetails.setExpiryMonth("08");
+        defaultPaymentMethodDetails.setExpiryYear("2020");
+        defaultPaymentMethodDetails.setHolderName("CARDHOLDER_NAME");
+        defaultPaymentMethodDetails.setNumber("666666xxxxxx6666");
```
Comment (smic1909) [inline src/test/java/com/adyen/CheckoutTest.java:1318]: Wouldn´t it be more consistent to you a MC masked cardbin?

File: src/test/java/com/adyen/CheckoutTest.java [hunk 2]
```diff
@@ -1288,6 +1301,33 @@ protected PaymentsRequest createEncryptedPaymentsCheckoutRequest() {
...
+     * Returns a sample PaymentsRequest object with type networkToken and brand visa
+     */
+    protected PaymentsRequest createPaymentsNetworkCheckoutRequest() {
+        PaymentsRequest paymentsRequest = new PaymentsRequest();
+
+        paymentsRequest.setReference("Your order number");
+        paymentsRequest.setAmount(createAmountObject("USD", 1000L));
+        DefaultPaymentMethodDetails defaultPaymentMethodDetails = new DefaultPaymentMethodDetails();
+        defaultPaymentMethodDetails.setType("networkToken");
+        defaultPaymentMethodDetails.setBrand("visa");
```
Comment (smic1909) [inline src/test/java/com/adyen/CheckoutTest.java:1314]: Wouldn´t it be more consistent to set the brand to mastercard?

File: src/test/java/com/adyen/CheckoutTest.java (hunk 1 above)
Comment (Aleffio) [reply src/test/java/com/adyen/CheckoutTest.java]: thanks, I replaced it 👍

File: src/test/java/com/adyen/CheckoutTest.java (hunk 2 above)
Comment (Aleffio) [reply src/test/java/com/adyen/CheckoutTest.java]: thanks, I replaced it 👍

**Pull request merged**